drivers/sony/displays/bravia.cr

```diff
@@ -137,12 +137,12 @@ class Sony::Displays::Bravia < PlaceOS::Driver
     param = data[7..-1]
 
     return task.try(&.abort("error")) if param.first? == ERROR
-    type = MessageType.from_value(data[2]).to_i64.chr
-    case type
-    when 'A'
+
+    case MessageType.from_value(data[2])
```

Comment:
Actually, `from_value?` is better here. Then logging `data[2]` in the else clause below would be a good idea if we want to diagnose malformed messages